src/github/pullRequestOverview.ts

```diff
@@ -43,16 +43,19 @@ export class PullRequestOverviewPanel {
 	private _initialized: boolean;
 	private _scrollPosition = { x: 0, y: 0 };
 
-	public static createOrShow(extensionPath: string, pullRequestManager: IPullRequestManager, pullRequestModel: IPullRequestModel) {
-		const column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+	public static createOrShow(extensionPath: string, pullRequestManager: IPullRequestManager, pullRequestModel: IPullRequestModel, toTheSide: Boolean = false) {
+		let activeColumn = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+		if (toTheSide && activeColumn) {
```

Comment:
this is kind of a weird case, but when I first tried this, I had the welcome page open instead of an editor, and the webview opened as normal instead of to the side of that. should we always set the `activeColumn` to beside when `toTheSide` is set?